src/lib/components/Access/utils.js

```diff
@@ -0,0 +1,149 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2020-2021 CERN.
+// Copyright (C) 2020-2021 Northwestern University.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+import React, { Component } from 'react';
+import { DateTime } from 'luxon';
+import { Button, Checkbox, Divider, Icon, List, Message } from 'semantic-ui-react';
+import {
+  FieldLabel,
+  TextAreaField,
+  TextField,
+} from 'react-invenio-forms';
+
+import { Embargo } from './Embargo';
+
+/**
+ * Returns the props for a protection button.
+ * @param active is button active
+ * @param activeColor button color when active
+ */
+export function getProtectionButtonProps(active, activeColor) {
+  let props = {active};
+  if (active) {
+    props["color"] = activeColor;
+  }
+  return props;
+}
+
+
+function ProtectionButtons({active}) {
+  return (
+    <Button.Group widths={'2'}>
+      <Button {...getProtectionButtonProps(active, "green")} compact attached>Public</Button>
+      <Button {...getProtectionButtonProps(!active, "red")} compact attached>Restricted</Button>
+    </Button.Group>
+  );
+}
+
+export function metadataSection(metadataPublic) {
```

Comment:
Maybe convert this to a functional component i.e something like `export const MetadataSection = () => {}`. Then, it's clear that this is a component rather than a helper function that returns a jsx.... Maybe I am thinking of reusable components instead of reusable functions....